### ISSUE TITLE
brush size in paint tool while painting different between tablet and trackpad

### ISSUE THREAD

**Issue by nithinshoban** [mbrlabs/Lorien#194]
**Lorien version:**

Loriean V 0.5.0

**OS/device including version:**

Windows Laptop,Wacom Intuos tablet

**Issue description:**

1.Open a new tab
2.Select Paint tool
3.Set pen size to 50
4.Draw using tablet
5.Draw using trackpad
6.Line thickness is different between tablet and trackpad

**Steps to reproduce:**
![image](https://user-images.githubusercontent.com/107873679/184099643-c29f2d5b-2a72-438d-9445-23137d5c1bfc.png)

**Comment by mbrlabs** [comment]
Yes this is probably a pressure sensitivity thing. When using the mouse/trackpad the pressure is always 100%, so lines are always full width. With a tablet you have variable width based on how hard you press.

**Comment by DampfLoque** [comment]
I have the same issue with an XP Pen 24 Screen Tablet, the pressure value doesn't change at all. It is fixed to the value I set in the Settings menu.

**Comment by wixkedbruixes** [comment]
@DampfLoque 

Same issue here, but with an XP-Pen Artist 22 Plus tablet. I just opened an issue on it -- fingers crossed it gets addressed soon! :)
